<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDlg</class>
 <widget class="QWidget" name="ChatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#tbut_inform{
	border:none;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_name">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>名字</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tbut_inform">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget_chat">
           <property name="styleSheet">
            <string notr="true">#listWidget_chat{
	border:none;
	border-top:2px solid rgb(231, 231, 231);
	
	background-color: rgb(245, 245, 245);
}
#listWidget_chat::item:hover{
	background-color: rgba(0, 0, 0, 0);
}
#listWidget_chat::item:selected{
	background-color: rgba(0, 0, 0, 0);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>175</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>175</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#widget_3{
	border-top:2px solid rgb(231, 231, 231);
	background-color: rgb(245, 245, 245);
}

#tbut_face{
	border:none;
	border-image: url(:/imageSource/face_default.png);
}
#tbut_face:hover{
	border:none;
	border-image: url(:/imageSource/face_hover.png);
}
#tbut_file{
	border:none;
	border-image: url(:/imageSource/file_default.png);
}
#tbut_face:file{
	border:none;
	border-image: url(:/imageSource/file_hover.png);
}
#tbut_cut{
	border:none;
	border-image: url(:/imageSource/cut_default.png);
}
#tbut_cut:hover{
	border:none;
	border-image: url(:/imageSource/cut_hover.png);
}
#tbut_log{
	border:none;
	border-image: url(:/imageSource/chatLog_default.png);
}
#tbut_log:hover{
	border:none;
	border-image: url(:/imageSource/chatLog_hover.png);
}
#tbut_voice{
	border:none;
	border-image: url(:/imageSource/voice_default.png);
}
#tbut_voice:hover{
	border:none;
	border-image: url(:/imageSource/voice_hover.png);
}
#tbut_video{
	border:none;
	border-image: url(:/imageSource/video_default.png);
}
#tbut_video:hover{
	border:none;

	border-image: url(:/imageSource/video_hover.png);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="tbut_face">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_file">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_cut">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_log">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_voice">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_video">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextEdit" name="textEdit_message">
                 <property name="styleSheet">
                  <string notr="true">#textEdit_message{
	border:none;
	background-color: rgb(245, 245, 245);
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_5" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#but_send{
	border-radius:5px;
	background-color: rgb(233, 233, 233);
	border:none;
	color: rgb(7, 193, 96);
}
#but_send:hover{
	border-radius:5px;
	background-color: rgb(210, 210, 210);
	border:none;
	color: rgb(7, 193, 96);
}
#widget_5{
	background-color: rgb(245, 245, 245);
}</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>50</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="but_send">
                    <property name="minimumSize">
                     <size>
                      <width>128</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>128</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>发送</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
